audformat/core/scheme.py

```diff
@@ -200,12 +202,14 @@ def to_pandas_dtype(self) -> typing.Union[
             return pd.api.types.CategoricalDtype(
                 categories=labels, ordered=False,
             )
+        elif self.dtype == define.DataType.BOOL:
+            return 'boolean'
```

Comment:
Here's the important bit where we set `dtype='boolean'`, which is nullable. Other changes here are just cosmetic.